src/main/cpp/VideoState.h

```diff
@@ -72,6 +72,11 @@ extern "C" {
 // fixed point to double
 #define CONV_FP(x) ((double) (x)) / (1 << 16)
 
+#define ENABLE_SHOW_STATUS 1
+#define ENABLE_FAST_DECODE 0
+// generate missing pts for audio if it means parsing future frames
+#define ENABLE_GENERATE_PTS 0
```

Comment:
Same here (see VideoState.h comment on line 75)

Reply:
Changed to const